resources/data/elephpants.json

```diff
@@ -458,9 +458,9 @@
         },
         {
             "id": 58,
-            "name": "PHPDay",
+            "name": "Aida",
```

Comment:
The right name: `phpday`

Reply:
Giulia confirmed that the name is `Aida`. But yours spelling is correct, so I've ported it into the description, as per Fullo's suggestions.